Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```diff
@@ -300,3 +300,6 @@
     containers:
       mech-battery-slot:
       - PowerCellHigh
+  - type: Access
+    tags:
+    - Maintenance
```

Comment:
You should move this to MechVim, since MechVimBattery is an admin spawn only. Also please add media to your PR description showing the change in-game

Reply:
Working on the media, was seeing if there was a way to allow its access to be edited.

Reply:
I was thinking that the MechVimBattery was just the VIM with battery added to it. That way an unpowered VIM would not be able to open maintenance doors. Added media, updated description of what was changed.